### PR TITLE
Fixes #144 - Added the Redis URL to env.example and pass it when making Bull queue

### LINKED ISSUE #144: Specify redis url in our config
Split off from #112, this deals with specifying server info about redis in our env, and reading that in when we create a queue.

We should move the Redis URL into our .env and when we make Bull Queues, pass this URL on via the constructor. With #105, we can do this. Doing so will make it easier to switch between development and production environments:

```js
const redisUrl = process.env.REDIS_URL || 'redis://127.0.0.1:6379';
const queue = new Queue('name-of-queue', redisUrl);
```

### PR DESCRIPTION
**Pull request by zufishanali**
Fixes #144 

Changes made:
- Added the Redis URL to `env.example` 
- Pass the URL when making Bull queue via the constructor in `src/feed-queue.js`

### CODE REVIEW

File: env.example [hunk 1]
```diff
@@ -3,3 +3,6 @@ NODE_ENV=development
 
 # PORT is the port used by the web server
 PORT=8080
+
+# REDIS_URL specifies Redis server info
+REDIS_URL='redis://127.0.0.1:6379'
```
Comment (s-arika) [inline env.example]: Let's remove the quotes so that it'll be REDIS_URL=redis://127.0.0.1:6379 and add a new line after

File: src/feed-queue.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
+const config = require('./config');
```
Comment (s-arika) [inline src/feed-queue.js]: Put this after bull require and remove const config. Lines 1 and 2 will look like 
const Bull = require('bull');
require('./config');

File: src/feed-queue.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
 const Bull = require('bull');
 
-const queue = new Bull('feed-queue');
+const redisUrl = process.env.REDIS_URL;
```
Comment (s-arika) [inline src/feed-queue.js]: this line can be removed and replace line 5 with 
const queue = new Bull('feed-queue', process.env.REDIS_URL);

File: env.example (hunk 1 above)
Comment (zufishanali) [reply env.example]: Removed the quotes an added new line.

File: src/feed-queue.js (hunk 2 above)
Comment (zufishanali) [reply src/feed-queue.js]: Removed `const config = ` and moved it to be below bull require.

File: src/feed-queue.js (hunk 3 above)
Comment (zufishanali) [reply src/feed-queue.js]: Combined the lines to appear as requested.

**Pull request merged**